### DISCUSSION TITLE
Hi all!

### DISCUSSION THREAD

**Discussion by LaurenShone** [Green-Software-Foundation/discussions#118] [Introductions]
Hi everyone,

My name is Lauren Shone, I am a user experience/product designer and researcher. I am currently working for Accedo TV's Sustainability team, improving video-centric experiences while reducing environmental impact and helping customers and partners reach their sustainability goals.

I have found this space through the need for a community, as sustainable goals require collaboration and efficient communication. I hope to find here, not only fellow supporters of the GSF mission, but also those working within the video streaming space, TV apps etc. to help build our knowledge around sustainable streaming solutions, as well as potentially partner and help bring these solutions to life!

My LinkedIn:
https://www.linkedin.com/in/laurenshone/

Whitepaper, Exploring Sustainability in Video Applications:
https://www.accedo.tv/insights-and-news/white-paper-exploring-sustainability-in-video-applications

**Comment by pdematos12** [comment]
Hi @LaurenShone - you may want to get involved in a new project we starting.
Sign up details here: https://github.com/Green-Software-Foundation/patterns/discussions/309
